Sources/Fuzzilli/FuzzIL/AbstractInterpreter.swift

```diff
@@ -505,6 +508,11 @@ public struct AbstractInterpreter {
         case let op as LoadSuperProperty:
             set(instr.output, inferPropertyType(of: op.propertyName, on: currentSuperType()))
 
+        /* 
+        TODO: Add missing case for StoreSuperPoperty
```

Comment:
What about this? :)

Reply:
So I've been thinking of rewriting the ClassDef building following the same model that I've proposed for ObjectDef building. I'm not sure if there's a trivial way to add AI rules for StoreSuper ops (tbh I haven't given this a great deal of thought) but if you've got some suggestions then I'll look into it.

Reply:
Right, yeah I don't think updating the type of superclass properties is easily possible in the current model. So let's just remove these comments or turn them into `TODO support superclass property assignment` and deal with class definitions and superclass properties separately, wdyt?

Reply:
I think I'll turn this into a TODO so that we don't lose track of this piece of work.